tests/Struct/FindologicProductTest.php

```diff
@@ -390,13 +396,33 @@ public function testProduct(): void
             $config
         );
 
+        $productKeyword = new Keyword('FINDOLOGIC Keyword');
+        $ignoreKeyords = [
+            $productEntity->get('id'),
+            $productEntity->getProductNumber(),
+            $productEntity->getManufacturer()->getName()];
+
+        $keywords = $findologicProduct->getKeywords();
+        $expectedProductKeywordExists = false;
+        $ignoreProductKeywordExists = false;
+        $this->assertNotCount(0, $keywords);
+        foreach ($keywords as $keyword) {
+            if ($productKeyword->getValue() === $keyword->getValue()) {
+                $expectedProductKeywordExists = true;
+            }
+            if (in_array($keyword->getValue(), $ignoreKeyords)) {
+                $ignoreProductKeywordExists = true;
+            }
+        }
```

Comment:
As we are now manually ignoring the unwanted keywords here, I think we can directly test for the keywords array instead of checking for just one value inside of it. This will help us in future if some other keyword is also added by Shopware then it will fail here.

This way we can ensure only our configured keywords are being sent in export. Otherwise if another keyword is added we would not know as this test would still pass. 

Also add multiple keywords for that case so there is more than one value to check.